packages/gatsby-theme/src/layouts/index.js

```diff
@@ -44,26 +30,33 @@ const Thumb = props => (
 
 export default props => (
   <Root>
-    <div
+    <Header {...props} />
+    <ul
       css={{
-        padding: 32,
+        listStyle: 'none',
+        padding: 0,
+        display: 'flex',
+        flexWrap: 'wrap',
       }}
     >
-      <h1>MDX Decks</h1>
-      <ul
-        css={{
-          listStyle: 'none',
-          padding: 0,
-          display: 'flex',
-          flexWrap: 'wrap',
-        }}
-      >
-        {props.decks.map(deck => (
-          <li key={deck.id} style={{ width: 100 / 3 + '%' }}>
-            <Thumb {...props} {...deck} />
-          </li>
-        ))}
-      </ul>
-    </div>
+      {props.decks.map(deck => (
+        <li
+          key={deck.id}
+          css={{
+            width: '100%',
+            padding: 32,
+            '@media screen and (min-width: 48em)': {
+              width: '50%',
+            },
+            '@media screen and (min-width: 56em)': {
```

Comment:
These kind of selectors prevent consistent look among various window sizes and print view. I'm sure "adaptive" CSS must be opt-in.